pkg/interface/src/views/components/RemoteContent/index.tsx

```diff
@@ -125,9 +128,11 @@ function RemoteContentInner(props: RemoteContentProps) {
     );
   } else if (isOembed && remoteContentPolicy.oembedShown) {
     return (
-      <AsyncFallback fallback={fallback}>
-        <RemoteContentOembed ref={embedRef} url={url} renderUrl={renderUrl} oembed={oembed} />
-      </AsyncFallback>
+      <Box mt={1} mb={2} flexShrink={0}>
+        <AsyncFallback fallback={fallback}>
+          <RemoteContentOembed ref={embedRef} url={url} renderUrl={renderUrl} />
```

Comment:
@rcrdlbl I think you missed `oembed={oembed}` in the props for this merge, but after you commit that this is good to merge!

Reply:
Just pushed this commit!